### PR TITLE
Testing remaining Journal pages

### PR DESCRIPTION
**Pull request by giorgiosironi**
- Various semi-static pages
- Consistent testing of all known listings
- Parameterizing tests for independence

### CODE REVIEW

File: spectrum/checks.py [hunk 1]
```diff
@@ -503,8 +503,17 @@ def generic(self, path):
         LOGGER.info("Loading %s", url)
         response = requests.get(url)
         _assert_status_code(response, 200, url)
+        _assert_all_resources_of_page_load(response.content, self._host)
         return response.content
 
+    def listing(self, path):
+        body = self.generic(path)
+        soup = BeautifulSoup(body, "html.parser")
+        teaser_a_tags = soup.select("div.teaser h4 a")
```
Comment (thewilkybarkid) [inline spectrum/checks.py]: `.teaser__header_text_link` rather than `h4 a`?

Comment (giorgiosironi) [inline spectrum/checks.py]: Would be nice to make it more precise. Can you confirm:
- it's only going to match a single link per each of the teasers
- it works both with grids (e.g. labs experiments) and listings (e.g. collections)

Comment (thewilkybarkid) [inline spectrum/checks.py]: 👍

File: spectrum/test_journal.py [hunk 2]
```diff
@@ -2,6 +2,33 @@
...
+    "/community",
+    "/contact",
+    "/for-the-press",
+    "/resources",
+    "/terms",
+    #/who-we-work-with
+]
+
+LISTING_PATHS = [
+    '/articles/correction',
```
Comment (thewilkybarkid) [inline spectrum/test_journal.py]: `/annual-reports` was merged this morning.

**Pull request merged**